typos.toml

```diff
@@ -25,6 +25,7 @@ extend-exclude = [
 # Match Whole Word - Case Sensitive
 [default.extend-identifiers]
 ser = "ser" # ron::ser - Serializer
+sme = "SME" # Subject Matter Expert
```

Comment:
This is failing CI still: I'm not sure what the correct incantation is.

Reply:
What are you trying to do in the first place? The configuration you have there in `typos.toml` is basically saying that any identifier or variable that contains the lowercase the linear sequences of characters `sme` should have that `sme` uppercased to `SME`. (Typos is intended for source code and all that).

Reply:
I want it to recognize "SME" as not a typo.

Reply:
Then it would be  `SME = "SME"` I think. It's case sensitive and all that.

Reply:
```suggestion
SME = "SME" # Subject Matter Expert
```